### DISCUSSION TITLE
Project board

### DISCUSSION THREAD

**Discussion by extreme4all** [fastapi/fastapi#5742] [Questions] [answered]
a project board allows tracking of issues, PR's, you can have multiple board for certain topics or for specific projects, this can even have some automation to move issues, PR's accoriding to their state (TODO, in progress, in review, done, ...).

for example this can be used to track the state of all issues & PR's related to translation.

There is no one best way of using the project boards but here is a suggestion:
- New: all new issues, there are not yet refined and suited to work on
- To do: refined issues where the work, issue & expectations are clearly defined (the "backlog")
- Stuck: the person that started to work on the issue is stuck & needs help (label help wanted)
- In progress: someone has started to work on the issue (Draft PR opened)
- In review: PR opened, the maintainer needs to review the issue
- Done: merged

**Comment by tiangolo** [comment, marked as answer]
Thanks for the interest. But as I'm still doing most of the work required to maintain FastAPI, having another tool to manage and do more admin would mainly mean more work for me. :sweat_smile: But thanks for the suggestion. Maybe if in some future I feel like it would help me I'll use that.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
